### ISSUE TITLE
Alibaba cloud

### ISSUE THREAD

**Issue by mrkpl125** [yaronelh/False-Positive-Center#104]
Here is another email to submit FP to Alibaba cloud: antivirus@alibabacloud.com . It was given to us by their tech support so they recommend to send FPs to this email address.

**Comment by yaronelh** [comment]
I just searched for it on the web but couldn't find any mention of that email being part of their contact information.
It would be great if you can upload a redacted screenshot of their new instructions on how to contact their false positive team, from the email they've sent you, or another source. Cheers

**Comment by mrkpl125** [comment]
Of course. It was via their ticketing platform that we recieved a reply from them that mentions the correct email address to send FPs. Pls, see the attached screenshot. 
![Screenshot 2024-11-12 143203](https://github.com/user-attachments/assets/622766ae-b9fc-49e3-8959-d6eff67853ad)

**Comment by mrkpl125** [comment]
By the way, a false positive detection on our program was lifted the next day after we sent an FP to that email address. Whereas previously we couldn't get any response.

**Comment by yaronelh** [comment]
Excellent. Added 👍
Thanks
